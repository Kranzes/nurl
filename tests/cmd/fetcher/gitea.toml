args = ["https://codeberg.org/nsxiv/nsxiv", "v30"]
