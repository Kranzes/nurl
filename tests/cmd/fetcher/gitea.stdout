fetchFromGitea {
  domain = "codeberg.org";
  owner = "nsxiv";
  repo = "nsxiv";
  rev = "v30";
  hash = "sha256-swzTdQ6ow1At4bKRORqz6fb0Ej92yU9rlI/OgcinPu4=";
}